source/gent/tier1_hortense.rst

```diff
@@ -216,28 +221,97 @@ This storage space can be addressed with the environment variable $TMPDIR
-*******************
+
+.. _hortense_scratch_storage_quota_usage:
+
+Scratch storage quota usage
+***************************
 
 * You can check personal and project storage quota usage by running the ``my_dodrio_quota`` command.
 * If you want to check storage quota for specific projects, or for projects that are not listed automatically, use the ``-p`` option.
 * For a list of all options, run ``my_dodrio_quota -h``.
 
-*(more information soon)*
+
+.. _hortense_accessing_data_readonly:
+
+Accessing data via ``/readonly``
+********************************
+
+Due to the fairly aggressive page cache purging policy that is used by the `Lustre <https://www.lustre.org>`_
+storage software that is used for the Tier-1 Hortense scratch filesystem, you may need to make some changes
+to how you access data in your job scripts to avoid performance problems.
+
+Whether or not this is required depends on the specific characteristics of your workloads:
+the number of files that are read, how large those files are, how the data in these files is accessed
+(the I/O pattern), etc. Note that this applies to both input data for your workloads, as well as
+any software you have installed on the Tier-1 Hortense scratch filesystem (see also :ref:`hortense_software_readonly`).
+
+To mitigate performance problems, you can access the data in your project scratch directory
+through the ``/readonly`` mount point, rather than accessing it directly.
+**We recommend to access your data via the** ``/readonly`` **mount point whenever possible,
+since it can result in a signficant performance speedup for your jobs.**
+
+This is done by prefixing the path to files and directories with ``/readonly/`` in your job script:
+rather than accessing your data via ``$VSC_SCRATCH_PROJECTS_BASE/...`` (or ``/dodrio/scratch/...``,
+which you should not used), you just use ``/readonly/$VSC_SCRATCH_PROJECTS_BASE/...`` instead.
+For example:
+
+.. code:: shell
+
+   export INPUT_DATA=/readonly/VSC_SCRATCH_PROJECTS_BASE/your_project_name/inputs/
+   python example_process_data.py $INPUT_DATA
+
+
+As the name suggests, the ``/readonly`` mount point only provides *read-only* access to your data.
+Trying to make any changes to files accessing via ``/readonly`` will result in ``Permission denied`` errors.
+
+.. note::
+
+   On the login nodes, there is a delay of maximum 30 minutes for changes to files (or new/removed
+   files/directories) to be reflected through the ``/readlonly/`` mount point.
+
+   In jobs, any changes you make to files or directories in your project scratch directory should be reflected
+   through the ``/readonly`` mount point immediately, as long as the job started running *after* the
+   changes were made.
+
+   In addition, take into account that changes which are made while the job is running will *not* be
+   reflected through the ``/readonly`` mount point (for that job).
+   If your job script creates new files, updates existing files, etc., those changes will *not* be
```

Comment:
new files/dirs might show up; but people should not assume so. not sure how to phrase this. i don;t mind making it thi strict, but it's not 100% true